server/src/data/seeds/fill-database.js

```diff
@@ -40,19 +43,19 @@ export async function seed(knex) {
         .whereIn('link', mapLinks(postImagesSeed));
 
       // Add users.
-      const usersMappedSeed = usersSeed.map((user, idx) => ({
+      const usersMappedSeed = usersSeed.map((user, index) => ({
         ...user,
-        [ColumnName.IMAGE_ID]: userImages[idx] ? userImages[idx].id : null
+        [ColumnName.IMAGE_ID]: userImages[index] ? userImages[index].id : null
       }));
       const users = await trx(TableName.USERS)
         .insert(usersMappedSeed)
         .returning('*');
 
       // Add posts.
-      const postsMappedSeed = postsSeed.map((post, idx) => ({
+      const postsMappedSeed = postsSeed.map((post, index) => ({
```

Comment:
Just for future, just create a new task for code refactor)

Reply:
It's a part of the linter rule. It forces to use full word instead of shortcuts